app/main/views/users.py

```diff
@@ -18,17 +18,27 @@
 def find_user_by_email_address():
     template = "view_users.html"
     users = None
+    teammembers = None
 
     email_address = request.args.get("email_address", None)
     if email_address:
-        users = data_api_client.get_user(email_address=request.args.get("email_address"))
+        users = data_api_client.get_user(email_address=email_address)
+
+    if users is None and email_address and "@" not in email_address:
```

Comment:
why the check on `@`?

Reply:
to prevent calling of team members. that particular api method searches in the domain field where there is no @.

Reply:
this may not be obvious to the user - may need to tell them thats how it works

Reply:
i'll add a comment to to the code and update the example of the field